indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -605,12 +605,14 @@ def get_track_reviewing_state(self, track):
     def get_track_question_scores(self):
         query = (db.session.query(AbstractReview.track_id,
                                   AbstractReviewQuestion,
-                                  db.func.avg(AbstractReviewRating.value))
+                                  db.func.avg(AbstractReviewRating.value.op('#>>')('{}').cast(db.Integer)))
                  .join(AbstractReviewRating.review)
                  .join(AbstractReviewRating.question)
                  .filter(AbstractReview.abstract == self,
+                         AbstractReviewQuestion.field_type == 'rating',
+                         AbstractReviewRating.value.cast(db.String) != db.text("'null'"),
```

Comment:
I'd still change the field type to JSONB and use `jsonb_typeof` for the check.

Reply:
why jsonb?

Reply:
Because there is no `json_typeof`. `JSON` is stored as a string, while `JSONB` is stored in a binary format that allows postgres to actually look at its contents.

Reply:
are you sure there is no `json_typeof`? https://www.postgresql.org/docs/9.6/static/functions-json.html

Reply:
Indeed, looks like it's only the operators that are JSONB-only. Anyway, the performance is probably better with JSONB

Reply:
Yeah, but do we need that performance in a field which holds only one simple value?

Reply:
No, it probably doesn't make a noticeable difference here.